packages/editor/src/components/provider/index.native.js

```diff
@@ -158,6 +160,7 @@ export default compose( [
 	} ),
 	withDispatch( ( dispatch ) => {
 		const {
+			clearSelectedBlock,
```

Comment:
It looks like `clearSelectedBlock` from `core/editor` just [calls through](https://github.com/WordPress/gutenberg/blob/master/packages%2Feditor%2Fsrc%2Fstore%2Factions.js#L1021) to `core/block-editor` with a deprecation warning that shows up as a Yellow Box warning. Seems like we should just get this directly from `core/block-editor` instead.

Reply:
Thanks, updated 👍

Reply:
I think we also need to update `header-toolbar`: https://github.com/WordPress/gutenberg/pull/16906/files#diff-dbd7e14b9675d5446422c008453093bbR91